Source/Scene/BingMapsImageryProvider.js

```diff
@@ -574,7 +570,7 @@ define([
+    BingMapsImageryProvider._logoData = buildModuleUrl('Assets/Images/bing_maps_credit.png');
```

Comment:
CI is failing for `test node` because `buildModueUrl` calls `document.createElement('a')`.  Suggestions for fixing this?